test/library_search/filter.lean

```diff
@@ -5,9 +5,10 @@ open filter
 /- Turn off trace messages so they don't pollute the test build: -/
 set_option trace.silence_library_search true
 
-example {α β γ : Type*} {A : filter α} {B : filter β} {C : filter γ} {f : α → β} {g : β → γ}
-  (hf : tendsto f A B) (hg : tendsto g B C) : map (g ∘ f) A = map g (map f A) :=
-by library_search
+--CI timeouts here, but `library_search` correctly finds `map_map.symm`
+-- example {α β γ : Type*} {A : filter α} {B : filter β} {C : filter γ} {f : α → β} {g : β → γ}
+--   (hf : tendsto f A B) (hg : tendsto g B C) : map (g ∘ f) A = map g (map f A) :=
+-- by library_search
```

Comment:
Does using `try_for 200000 { library_search }` work here?

Reply:
It seems it does, thanks!